karusc.client/src/pages/ProductForm.tsx

```diff
@@ -0,0 +1,162 @@
+// ProductForm.tsx
+import React, { useState, useRef } from 'react';
+import { Form, Button, Card, Col, Row, Modal } from 'react-bootstrap';
+
+interface ProductFormProps {
+  onAddProduct: (product: { productName: string, productCategory: string, productDescription: string, images: string[] }) => void;
+}
+
+const ProductForm: React.FC<ProductFormProps> = ({ onAddProduct }) => {
+  const [images, setImages] = useState<string[]>([]);
```

Comment:
Use existing Product model and make appropriate changes as required

Reply:
Used the Product Model in ProductForm page